scripts/lib/CIME/SystemTests/system_tests_compare_two.py

```diff
@@ -36,10 +47,11 @@ class SystemTestsCompareTwo(SystemTestsCommon):
 
     def __init__(self,
                  case,
-                 separate_builds,
+                 separate_builds=True,
```

Comment:
I'd prefer if this remained a required argument so that test developers have to actively think of whether this should be True or False. (I was okay with @jgfouca changing it in his PR for symmetry with separate_run_dirs, but now that separate_run_dirs is gone, I'd prefer if this remained as it was before.)